platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -619,7 +619,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.
+ like {field_name}.  (Token replacement is only supported for literal text-field
+ values--not for data-driven property functions.)
```

Comment:
Surround “text-field” in backticks. This allows generate-style-code.js to replace it with `text` (the overridden name of this property).